app-editors/vim/patches/vim-8.2.0111.patch

```diff
@@ -340,7 +340,7 @@ index 203276f64..3fcfec098 100644
 +
  Note that creating a vimrc file will cause the 'compatible' option to be off
  by default.  See |compatible-default|.
- 
```

Comment:
Please don't remove trailing whitespace from the patch. They are annotations for `patch` to differentiate between the end of the patch vs an empty line in the patch.

Reply:
Fixed!